presto-verifier/src/main/java/com/facebook/presto/verifier/framework/VerificationContext.java

```diff
@@ -20,26 +20,50 @@
 import java.util.Set;
 
 import static com.google.common.collect.ImmutableList.toImmutableList;
+import static java.util.Objects.requireNonNull;
 
 public class VerificationContext
 {
+    private final String sourceQueryName;
```

Comment:
Just double check: is this the original query id? :)

Reply:
Theoretically this could be anything provided in the xdb, but yes it is the original query ID for FB use cases.